src/controllers/handlers/about-handler.ts

```diff
@@ -0,0 +1,85 @@
+import { HandlerContextWithPath } from '../../types'
+
+export type About = {
+  healthy: boolean
+  configurations: {
+    commsProtocol: string
+  }
+  content: {
+    healthy: boolean
+  }
+  comms: {
+    healthy: boolean
+  }
+  lambdas: {
+    healthy: boolean
+  }
+}
+
+// handlers arguments only type what they need, to make unit testing easier
+export async function aboutHandler(
+  context: Pick<
+    HandlerContextWithPath<'serviceDiscovery' | 'logs' | 'metrics' | 'config' | 'fetch', '/explorer-configuration'>,
+    'url' | 'components'
+  >
+) {
+  const { logs, config, fetch, serviceDiscovery } = context.components
+
+  const logger = logs.getLogger('explorer-configuration')
+  const commsProtocol = await config.requireString('COMMS_PROTOCOL')
+  const lambdasUrl = await config.requireString('LAMBDAS_URL')
+
+  const body: About = {
+    healthy: false,
+    configurations: {
+      commsProtocol
+    },
+    content: {
+      healthy: false
+    },
+    comms: {
+      healthy: false
+    },
+    lambdas: {
+      healthy: false
+    }
+  }
+
+  try {
+    const response = await fetch.fetch(`${lambdasUrl}/health`, {
```

Comment:
should we check the response code here too?

Reply:
I don't think so, I added some unit tests